src/resmom/requests.c

```diff
@@ -757,16 +756,18 @@ message_job(job *pjob, enum job_file jft, char *text)
 	SetFilePointer((HANDLE)_get_osfhandle(fds), (LONG)NULL,
 		(PLONG)NULL, FILE_END);
 #else
-	while ((slept < 5) && (fds < 0)) {
+	int i;
+	unsigned int usecs = 250 * 1000; /* 250 milliseconds */
+	for (i = 0; i < 3; i++) {
 		fds = open_std_file(pjob, jft, O_WRONLY | O_APPEND | O_NONBLOCK,
 	                         pjob->ji_qs.ji_un.ji_momt.ji_exgid);
-		if (fds < 0) {
-			if (errno == EAGAIN || errno == EWOULDBLOCK) {
-				sleep(1);
-				slept++;
-			} else
+		if (fds > 0)
```

Comment:
Was this change intentional? If fd is > 0 then that would mean open() succeeded but we will still retry if errno is set to EAGAIN or EWOULDBLOCK.

Reply:
not intentional. I have pushed the correct changes now.